services/core-web/cypress/e2e/createproject.cy.ts

```diff
@@ -38,7 +38,6 @@ describe("Major Projects", () => {
 
     // SAVE & CONTINUE - skip to Purpose & Authorization
     cy.contains("Save & Continue").click({ force: true });
-    cy.contains("Project Lead", { timeout: 10000 });
```

Comment:
I couldn't find in Core an element with Project Lead in Major Project Summary section so I ended up removing this test that was causing an error.